src/Avalonia.OpenGL/Composition/ICompositionGlContext.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.Threading.Tasks;
+using Avalonia.Metadata;
+using Avalonia.Rendering.Composition;
+
+namespace Avalonia.OpenGL.Composition;
+
+[NotClientImplementable, Unstable]
+public interface ICompositionGlContext : IAsyncDisposable
```

Comment:
Do we need Unstable on these new APIs? It's going to be marked as obsolete. 
If there is a reason, it can be specified in the Unstable attr parameter.

Reply:
The API is more or less experimental, so I'd rather avoid setting those in stone before getting usage feedback